src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/conv_sum_broadcast.cpp

```diff
@@ -167,6 +167,23 @@ TEST_P(ConvSumInPlaceTest, CompareWithRefs) {
     CheckPluginRelatedResults(compiledModel, "Convolution");
 }
 
+class ConvSumInPlaceTest_FP16 : public ConvSumInPlaceTest {
```

Comment:
In this case we don't need the dedicated class. We can simply filter `cpu_f16_plugin_config` by ISA.

Reply:
the primTypeCheck is different from bf16 test case, so I still create ConvSumInPlaceTest_FP16 to inherit ConvSumInPlaceTest